src/graph.js

```diff
@@ -3,6 +3,38 @@ class Graph {
   constructor() {
     this.graph = {};
   }
+
+addNode(value) {
```

Comment:
Make sure your indentation is consistent throughout your code. All these methods should be indented once more so that someone reading this code can easily tell that these methods are all part of the class.